rules/src/main/scala/fix/UseNamedParameters.scala

```diff
@@ -100,8 +100,11 @@ final class UseNamedParameters(config: UseNamedParametersConfig)
                 case Some(symInfo)
                     // In the case of repeated parameters, if the parameter name is given only at the beginning, it is broken.
                     if !symInfo.signature.toString().startsWith("* Tuple") =>
-                  val paramName = Term.Name(symInfo.displayName).toString
-                  Patch.addLeft(t, s"$paramName = ")
+                  val pattern =  "[A-Za-z]\\d*".r
```

Comment:
Good idea to "lift" this out of the code as much as possible so we're not recompiling the regex ever time we run a match.

Reply:
Thank you very much! Correspondence.